content/en/docs/components/trainer/user-guides/local-execution-mode/_index.md

```diff
@@ -0,0 +1,6 @@
++++
+title = "Local Execution Mode"
```

Comment:
Shall we call it Execute TrainJobs Locally, like in KFP: https://www.kubeflow.org/docs/components/pipelines/user-guides/core-functions/execute-kfp-pipelines-locally/ ?